demo/src/lib/stackblitz/angular/index.html

```diff
@@ -4,7 +4,12 @@
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>AgnosUI angular stackblitz demo</title>
-		<!--	STYLE CONFIG -->
+		<link
+			href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+			rel="stylesheet"
+			integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+			crossorigin="anonymous"
+		/>
 	</head>
 	<body style="margin: 1rem; min-height: 100vh!">
```

Comment:
since we use different index files for Bootstrap and Tailwind, we can go back to use the Bootstrap utilities class